src/components/TodoCellCompleted.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon, Table } from 'semantic-ui-react';
+
+const TodoCellCompleted = ({ compl }) => (
+  <Table.Cell negative={!compl} positive={compl}>
+    <Icon
+      color={compl ? 'green' : 'red'}
```

Comment:
```suggestion
      color={complete ? 'green' : 'red'}
```
Шо за экономия 4 букв?

Reply:
это осталось с задания static-list:) исправлю